.NET/Microsoft.Recognizers.Text.DateTime/Constants.cs

```diff
@@ -121,6 +121,7 @@ public static class Constants
 
         public const string DefaultLanguageFallback_MDY = "MDY";

```

Comment:
The comment is missing here.

Reply:
Sorry for that, I will add it immediately.